api_v3/services/LiveStreamService.php

```diff
@@ -617,6 +618,46 @@ public function archiveAction($liveEntryId, $vodEntryId)
         return true;
     }
 
+	protected function responseHandlingGetStats($isLive)
+	{
+		$liveStatsInterval = kConf::get('liveStatsInterval','live');
```

Comment:
take live mape name from kConfMapNames class
also add default value of 5 for the interval

Reply:
didnt nadav asked for 20?

Reply:
yes it was before, put 20